### ISSUE TITLE
Mixins with semi colons not converted correctly

### ISSUE THREAD

**Issue by mosid** [brauliobo/less2sass#11]
less code 
`.border-radius(0; 0; @box-border-radius; @box-border-radius);`

is converted to this scss code
`@include border-radius(0; 0; $box-border-radius; $box-border-radius);`

semi colons causes errors in sass

**Comment by swrobel** [comment]
Pull requests are welcome! At least provide the correct syntax...

**Comment by swrobel** [comment]
Opened a SO thread about this but so far nothing seems to nail it: http://stackoverflow.com/q/39757272/337446

**Comment by mosid** [comment]
I am not good at ruby, but i think Wiktor Stribiżew answer is the closest one.
Did you try it?

**Comment by mohammed-alsiddeeq** [comment]
After i second look,
I think the question you posted in SO is a little misleading.

.mixin is NOT keyword in less
the Wiktor Stribiżew answer is grabing the . selector from less and converting it to sass
eg: if less mixin is 

``` css
 .whatever()
```

It is going to be converted to

``` css
 #whatever 
```

which obviously not a sass keyword

**Comment by jimryan** [comment]
I don't think this can be done in a single gsub pass as requested on SO - not with an unknown number of parameters, anyway.  Would you accept a PR that uses Procs as the `Replaces` hash values?  This would allow us to perform additional substitutions as necessary.  The mixin definition and invocation key/values would then become:

```
  '\.([\w\-]*)\s*\((.*)\)\s*\{' => proc { "@mixin #{$1}\(#{$2.gsub(/;/, ',')})\n{" },
  '\.([\w\-]*)\((.*)\)(\s*;)' => proc { line_end=$3; "@include #{$1}(#{$2.gsub(/;/, ',')})#{line_end}" },
```

Happy to submit a PR if you're willing to move in this direction.  Right now, I'd consider this awesome tool to be broken, as it does not produce a valid SCSS file if multi-argument mixins are used, so this does need to be addressed.
